src/test/rgw/test_rgw_bencode.cc

```diff
@@ -13,7 +13,8 @@ TEST(Bencode, Str)
   decode.bencode("bar", bl);
   decode.bencode("baz", bl);
 
-  ASSERT_EQ("3:foo3:bar3:baz", bl.c_str());
+  const char *p = "3:foo3:bar3:baz";
+  ASSERT_EQ(0, strcmp(p, bl.c_str()));
```

Comment:
oops, ASSERT_STREQ() is what you want for `const char*` comparison (see https://github.com/google/googletest/blob/master/googletest/docs/Primer.md)